packages/create-webstone-app/package.json

```diff
@@ -2,43 +2,45 @@
   "name": "create-webstone-app",
   "version": "0.7.0",
   "description": "Start your next web application with Webstone and configure it as you go.",
+  "keywords": [
+    "svelte",
+    "sveltekit",
+    "boilerplate",
+    "starterkit",
+    "web app",
+    "graphql"
+  ],
+  "license": "MIT",
   "author": "Mike Nikles, @mikenikles",
   "type": "module",
+  "exports": {
+    ".": "./dist/index.js"
+  },
+  "types": "./dist/index.d.ts",
```

Comment:
We probably also need to add `files` here: 

```json
{
"files": [
    "dist",
    "bin.js",
    "templates"
  ],
}
```

Not sure if I'm missing something here

Reply:
Good call, I think without a `files` property, everything is included in the package.